development/includes/_top-nav.pug

```diff
@@ -0,0 +1,15 @@
+nav(class="navbar navbar-default navbar-fixed-top")
+  div.container-fluid
+    div.navbar-header
+      button.navbar-toggle(type="button" data-toggle="collapse" data-target="#myNavbar")
+        span.icon-bar
+        span.icon-bar
+        span.icon-bar
```

Comment:
Let's delete all these spans and simply use one icon from the material icon set (Google's free icon set) https://design.google.com/icons/

`<i class="material-icons">menu</i>` 

To use this simply include add this to the head of your website layout 
```
<link href="https://fonts.googleapis.com/icon?family=Material+Icons"
      rel="stylesheet">
```